xmtp_mls/src/groups/mod.rs

```diff
@@ -214,7 +215,7 @@ where
         client: &'c Client<ApiClient>,
         membership_state: GroupMembershipState,
         permissions: Option<PreconfiguredPolicies>,
-        added_by_address: String,
+        added_by_address: &InboxId,
```

Comment:
Do you mind putting some TODOs on these cases where we are using inbox IDs as addresses, just so we can make sure to get to them all

Reply:
For sure, should've done that in the first place

Reply:
renamed this one to `inbox_id`, and commented a few `TODO` elsewhere